frontend/benefit/handler/src/utils/forms.ts

```diff
@@ -0,0 +1,29 @@
+import { FormikErrors, FormikTouched, FormikValues, getIn } from 'formik';
+import { TFunction } from 'next-i18next';
+
+/** Get error text
+ * @param {Object} errors
+ * @param {Object} touched
+ * @param {string} name
+ * @param {Function} t
+ * @param {boolean} isSubmitted
+ * @return {string}
+ */
+export const getErrorText = (
```

Comment:
Should we move this to the shared folder?

Reply:
its formik related, in ks we are not using formik, but probably will in handler. i would want to keep shared independent (one thing to refactor is to remove nextjs dependency from there at some point as well)

Reply:
Alright 👌